mdflynn/overlook
# Feature/bookings

## What is the change?
- Adds POST and Delete fetches
- Separates scss into pseudo files
- Designs delete and book buttons
- Adds a domUpdate object literal in a separate file
- Adds eventListeners for all buttons
- Adds bookings displays for both Manager and Guest
- Adds search option to Manager dashboard

## What does it fix?
No fix this PR.

## Is this a fix or a feature?
This is a feature.

## Where should the reviewer start?
- entire `api-request.js`
- entire `dom-update.js`
- entire `_mixins.scss`
- entire `_variables.scss`
- lines 46-146, 165-189, 207-351 in `index.js`

## How should this be tested?
Run `npm start` in your terminal. Then navigate to localhost:8080 in your browser to see site functionality.
